tiledb/ml/models/tensorflow_keras.py

```diff
@@ -15,7 +15,10 @@
 from tensorflow.python.keras import optimizer_v1
 from tensorflow.python.keras.saving import saving_utils
 from tensorflow.python.keras.saving.saved_model import json_utils
-
+from tensorflow.python.keras.utils import generic_utils
+from tensorflow.python.keras.saving import model_config as model_config_lib
+from tensorflow.python.keras import backend
+from tensorflow.python.keras.saving.hdf5_format import preprocess_weights_for_loading
```

Comment:
Actually the package name is quite misleading as that has nothing to do with `hdf5` format. The function is a recursive function that does some transformation in case of different keras formats. I can migrate the whole function in our codebase 
```
Preprocess layer weights between different Keras formats.
Converts layers weights from Keras 1 format to Keras 2 and also weights of
CuDNN layers in Keras 2.
```

Reply:
Why would you copy over a ~400 line function from a package that is already a dependency?

Reply:
Due to naming of the package 😛 , but as you said there is no point I reverted it!